libs/payments/stripe/src/lib/invoice.manager.ts

```diff
@@ -5,25 +5,19 @@
 import { Injectable } from '@nestjs/common';
 import { Stripe } from 'stripe';
 
-import { CustomerManager } from './customer.manager';
-import { SubscriptionManager } from './subscription.manager';
-
 import { StripeClient } from './stripe.client';
 import { StripeCustomer, StripeInvoice } from './stripe.client.types';
```

Comment:
Missing `type`? - `import type { StripeCustomer, StripeInvoice } from './stripe.client.types';`

Reply:
Didn't add these lines but it's also not really a diff since we're already importing the code and it's in-library. The type import vs the normal import don't make a functional difference in this case.